README.md

```diff
@@ -58,6 +58,55 @@ cd ../..
 ./tests/hmm/run.sh
 ~~~
 
+## Installation on Caltech HPC
+
+```bash
+#Go through the Caltech Tier2
+ssh USER@login-1.hep.caltech.edu
+
+#copy the skimmed samples to HPC, needs to be done once, or as top-up if you update the skim
+#use the rsync /./ and -R option to copy the full path structure, --ignore-existing to update just the new files
+rsync -rR --progress --ignore-existing /storage/user/nlu/./hmm/skim_merged USER@login1.hpc.caltech.edu:/central/groups/smaria/
+
+#now log in to the HPC machine
+ssh USER@login1.hpc.caltech.edu 
+
+#activate the prepared python environment (should you wish to do this from scratch, you can use the example in environments/setup-miniconda.sh)
+source /central/groups/smaria/jpata/miniconda3/bin/activate
+
+#make a working directory under the shared filesystem
+mkdir /central/groups/smaria/$USER
+cd /central/groups/smaria/$USER
+mkdir hmm
+cd hmm
+
+#get the code, compile the C++ helper library
+git clone https://github.com/jpata/hepaccelerate-cms
+git submodule init
+git submodule update
+cd hepaccelerate-cms/tests/hmm
+make -j4
+cd ../..
+
+#run a local test on the interactive login node and create jobfiles
+./tests/hmm/run_hpc.sh
+
```

Comment:
Please also add this `export SUBMIT_DIR=$SUBMIT_DIR` to the instructions. Thanks!